manifests/0000_70_cluster-network-operator_01_pki_crd.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: pkis.network.operator.openshift.io
+spec:
+  group: network.operator.openshift.io
```

Comment:
Are we allowed to define "subdomains" of `operator.openshift.io` here? That's not reserved for future use by `openshift/api`?

Reply:
Good question. @deads2k, do you have any thoughts about this?

Reply:
Good question. @deads2k, how do you feel about us using `network.operator.openshift.io` as a Group?